### ISSUE TITLE
Cannot pull files from blobs from python anymore

### ISSUE THREAD

**Issue by zepef** [Azure/azure-sdk-for-python#24482]
### Library name and version

azure-storage-blob==12.12.0

### Query/Question

Hello,

I cannot download blobs from python since azure-storage-blob==12.12.0.

def pull_blob_file(local_storage: str, blob_filename: str) -> str:

    blob_service_client_instance = BlobServiceClient(account_url=STORAGE_ACCOUNT_URL, credential=STORAGE_ACCOUNT_KEY)
    blob_client_instance = blob_service_client_instance.get_blob_client(BLOB_CONTAINER_NAME, blob_filename, snapshot=None)

    blob_data = blob_client_instance.download_blob()
    data = blob_data.readall()
    local_file = os.path.join(local_storage, blob_filename)
    print(local_file)
    with open(local_file, 'wb') as file:
        file.write(data)
        
    return local_file

### Environment

Python 3.8
VSCode

**Comment by YalinLi0312** [comment]
Thanks for your feedback, we'll investigate ASAP.

**Comment by Bogdan-Stancescu-RBC** [comment]
Same here. Code sample:

Shell:
```bash
export TEST_STORAGE="" # fill in a working storage account
```

Then, in Python:
```python
import os
from azure.storage.blob import BlobServiceClient
from azure.identity import DefaultAzureCredential

myStorage = bytes(os.environ["TEST_STORAGE"], "utf-8").decode(encoding="utf-8")
token_credential = DefaultAzureCredential()

blob_service_client = BlobServiceClient(
    account_url = f"https://{myStorage}.blob.core.windows.net",
    credential = token_credential
)

containers = blob_service_client.list_containers()
for containerMeta in containers:
    print("Container", containerMeta.name) # All good
    container_client = blob_service_client.get_container_client(containerMeta.name)
    for blobMeta in container_client.list_blobs():
        print(blobMeta.name) # azure.core.exceptions.HttpResponseError: This request is not authorized to perform this operation using this permission.
```

**Comment by vincenttran-msft** [comment]
Hi @zepef Pierre-Emmanuel, unfortunately I was unable to reproduce your error as it seems to be working on my end. I think the first step is to try to clear up any ambiguity that might be the source of the error.

1. What exactly are you failing at (line or API call), and what is the exact error? 
2. Just to confirm, you note that you cannot download "_since azure-storage-blob==12.12.0._". If I am reading that correctly, what was the last version of `azure-storage-blob` that the operation worked for you?
3. For `STORAGE_ACCOUNT_URL`, please ensure it is in the following format: `f"https://<storage_acc_name>.blob.core.windows.net"`
4. For `STORAGE_ACCOUNT_KEY`, please ensure the key is correct and is still valid for your Storage account. Perhaps try rotating the key and entering the new key?
5. `BLOB_CONTAINER_NAME` should be a container that already exists on the Storage account
6. `blob_filename` should be a blob that already exists in the provided container (`BLOB_CONTAINER_NAME`)

Hopefully double-checking those values should help you resolve your issue. Please let me know if you have any additional follow-ups, and if possible, please provide a `RequestID` along with your future query. Thanks!
**NOTE:** When providing a `RequestID`, please give a more recent request's ID as the `RequestIDs` are automatically cleared on a rolling basis.



Hi @Bogdan-Stancescu-RBC Bogdan, just as above, I was unable to reproduce your error as it seems to be working on my end. This seems to possibly be permission related, so please check your permissions (or consider using a different credential, if possible). 
As a side-note, I attempted to reproduce your issue with `azure-storage-blob=12.12.0` and `azure-identity=1.10.0`. 
Furthermore, please ensure that `myStorage` resolves to your Storage account name properly.
If none of that is able to help resolve your issue, please attach a `RequestID` just as requested above for the other poster's query. Thanks!

**Comment by Bogdan-Stancescu-RBC** [comment]
@vincenttran-msft I have actually attempted to reproduce the same issue with vanilla azure CLI, and it works when I try that. It's rather difficult for me to provide an abstract code sample that works in CLI and doesn't work in Python (notice the hoops I had to go through in the sample above). On a personal level, I can assure you my CLI user is really allowed to do pretty much anything with those blobs (I'm the owner), but I'd like us to find a way to replicate the issue without personal assurances. :)

**Comment by zepef** [comment]
Hello,

Thank You very much, I'll try all of that right away. I will keep You updated.

All the best,

Pierre-Emmanuel

De : vincenttran-msft ***@***.***>
Envoyé : mardi 17 mai 2022 23:29
À : Azure/azure-sdk-for-python ***@***.***>
Cc : Pierre-Emmanuel FEGA ***@***.***>; Mention ***@***.***>
Objet : Re: [Azure/azure-sdk-for-python] Cannot pull files from blobs from python anymore (Issue #24482)


Hi @zepef<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzepef&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kao%2BKgWcP%2BE4vRX8eKH874sZ8rYiAGLvKM9FpXWDnMc%3D&reserved=0> Pierre-Emmanuel, unfortunately I was unable to reproduce your error as it seems to be working on my end. I think the first step is to try to clear up any ambiguity that might be the source of the error.

  1.  What exactly are you failing at (line or API call), and what is the exact error?
  2.  Just to confirm, you note that you cannot download "since azure-storage-blob==12.12.0.". If I am reading that correctly, what was the last version of azure-storage-blob that the operation worked for you?
  3.  For STORAGE_ACCOUNT_URL, please ensure it is in the following format: f"https://<storage_acc_name>.blob.core.windows.net<https://%3cstorage_acc_name%3e.blob.core.windows.net>"
  4.  For STORAGE_ACCOUNT_KEY, please ensure the key is correct and is still valid for your Storage account. Perhaps try rotating the key and entering the new key?
  5.  BLOB_CONTAINER_NAME should be a container that already exists on the Storage account
  6.  blob_filename should be a blob that already exists in the provided container (BLOB_CONTAINER_NAME)

Hopefully double-checking those values should help you resolve your issue. Please let me know if you have any additional follow-ups, and if possible, please provide a RequestID along with your future query. Thanks!
NOTE: When providing a RequestID, please give a more recent request's ID as the RequestIDs are automatically cleared on a rolling basis.

Hi @Bogdan-Stancescu-RBC<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBogdan-Stancescu-RBC&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dU8oM8hEoXemhlUqQvxKtaJKSFJIc8GegbtLyuKQGwE%3D&reserved=0> Bogdan, just as above, I was unable to reproduce your error as it seems to be working on my end. This seems to possibly be permission related, so please check your permissions (or consider using a different credential, if possible).
As a side-note, I attempted to reproduce your issue with azure-storage-blob=12.12.0 and azure-identity=1.10.0.
Furthermore, please ensure that myStorage resolves to your Storage account name properly.
If none of that is able to help resolve your issue, please attach a RequestID just as requested above for the other poster's query. Thanks!

-

**Comment by zepef** [comment]
Hello again,


  1.  Here is the total error dump:

Traceback (most recent call last):
  File "e:\LocalRepos\Projects\systra-semantic-search\systrasemsearch\sem_search_cli.py", line 244, in <module>
    local_file = pull_blob_file(local_storage, document_to_retrieve)
  File "e:\LocalRepos\Projects\systra-semantic-search\systrasemsearch\sem_search_cli.py", line 178, in pull_blob_file
    blob_data = blob_client_instance.download_blob()
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\tracing\decorator.py", line 73, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_blob_client.py", line 868, in download_blob
    return StorageStreamDownloader(**options)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 355, in __init__
    self._response = self._initial_request()
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 435, in _initial_request
    process_storage_error(error)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_shared\response_handlers.py", line 181, in process_storage_error
    exec("raise error from None")   # pylint: disable=exec-used # nosec
  File "<string>", line 1, in <module>
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 392, in _initial_request
    location_mode, response = self._clients.blob.download(
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\tracing\decorator.py", line 73, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_generated\operations\_blob_operations.py", line 1677, in download
    map_error(status_code=response.status_code, response=response, error_map=error_map)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\exceptions.py", line 107, in map_error
    raise error
azure.core.exceptions.ResourceNotFoundError: The specified blob does not exist.
RequestId:4fc04976-c01e-001b-6164-6aa335000000
Time:2022-05-18T03:06:02.8609993Z
ErrorCode:BlobNotFound
Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:4fc04976-c01e-001b-6164-6aa335000000
Time:2022-05-18T03:06:02.8609993Z</Message></Error>


  1.  Last azure-storage-blob version that worked for me was 12.11.0


  1.  Putting f in front of correct STORAGE_ACCOUNT_URL value gives me: ValueError: Unable to determine account name for shared key credential. It not issuing this error without the added f.




  1.  I have rotated storage key which gives the same error



  1.  Container systracontainer does exist on storage account



  1.  Blob file exists in blob


Here is the CLI I/O:


  1.  There are 16 documents in the 'azureblob-index' search index.
  2.  <Response [200]>
  3.  Please enter a query in French or English:
What is a knowledge graph?

  1.  en
  2.  Status code:  200
  3.  Please enter the response number:
0

  1.  Creating a Knowledge Graph with neo4j.pdf
  2.  Error message


  1.  I get the number of files in index based on blob files imported
  2.  This show that connection is ok,
  3.  I type in my semantic query
  4.  English is detected as input language by ai-textanalytics,
  5.  connection to blob storage is validated again,
  6.  I choose first line of results dataframe,
  7.  the name of pdf file is successfully returned,
  8.  exit with error message when I try to pull blob file from Azure storage to local disk.

Code has not been altered since its delivered version to the client last Thursday.

Thank You for your help,

Pierre-Emmanuel

De : vincenttran-msft ***@***.***>
Envoyé : mardi 17 mai 2022 23:29
À : Azure/azure-sdk-for-python ***@***.***>
Cc : Pierre-Emmanuel FEGA ***@***.***>; Mention ***@***.***>
Objet : Re: [Azure/azure-sdk-for-python] Cannot pull files from blobs from python anymore (Issue #24482)


Hi @zepef<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzepef&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kao%2BKgWcP%2BE4vRX8eKH874sZ8rYiAGLvKM9FpXWDnMc%3D&reserved=0> Pierre-Emmanuel, unfortunately I was unable to reproduce your error as it seems to be working on my end. I think the first step is to try to clear up any ambiguity that might be the source of the error.

  1.  What exactly are you failing at (line or API call), and what is the exact error?
  2.  Just to confirm, you note that you cannot download "since azure-storage-blob==12.12.0.". If I am reading that correctly, what was the last version of azure-storage-blob that the operation worked for you?
  3.  For STORAGE_ACCOUNT_URL, please ensure it is in the following format: f"https://<storage_acc_name>.blob.core.windows.net<https://%3cstorage_acc_name%3e.blob.core.windows.net>"
  4.  For STORAGE_ACCOUNT_KEY, please ensure the key is correct and is still valid for your Storage account. Perhaps try rotating the key and entering the new key?
  5.  BLOB_CONTAINER_NAME should be a container that already exists on the Storage account
  6.  blob_filename should be a blob that already exists in the provided container (BLOB_CONTAINER_NAME)

Hopefully double-checking those values should help you resolve your issue. Please let me know if you have any additional follow-ups, and if possible, please provide a RequestID along with your future query. Thanks!
NOTE: When providing a RequestID, please give a more recent request's ID as the RequestIDs are automatically cleared on a rolling basis.

Hi @Bogdan-Stancescu-RBC<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBogdan-Stancescu-RBC&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dU8oM8hEoXemhlUqQvxKtaJKSFJIc8GegbtLyuKQGwE%3D&reserved=0> Bogdan, just as above, I was unable to reproduce your error as it seems to be working on my end. This seems to possibly be permission related, so please check your permissions (or consider using a different credential, if possible).
As a side-note, I attempted to reproduce your issue with azure-storage-blob=12.12.0 and azure-identity=1.10.0.
Furthermore, please ensure that myStorage resolves to your Storage account name properly.
If none of that is able to help resolve your issue, please attach a RequestID just as requested above for the other poster's query. Thanks!

-

**Comment by zepef** [comment]
Hello again,

Now I am trying:

def pull_blob_file(local_storage: str, blob_filename: str) -> str:

    local_file = os.path.join(local_storage, blob_filename)
    print(local_file, blob_filename)

    blob = BlobClient.from_connection_string(conn_str="DefaultEndpointsProtocol=https;AccountName=systrastore;AccountKey=<real_account_key_obfuscated_but rotated>EndpointSuffix=core.windows.net", container_name="systracontainer", blob_name=blob_filename)

    with open(local_file, "wb") as my_blob:
        blob_data = blob.download_blob()
        blob_data.readinto(my_blob)

Trying to access by connection string and putting litteral conn_str and container_name doesn't work either even if output is:

There are 16 documents in the 'azureblob-index' search index.
<Response [200]>
Please enter a query in French or English:
What is a knowledge graph?
en
Status code:  200
Please enter the response number:
0
Creating a Knowledge Graph with neo4j.pdf

>>> result of print(local_file, blob_filename)
C:\Users\Public\Documents\Creating a Knowledge Graph with neo4j.pdf Creating a Knowledge Graph with neo4j.pdf

And then:

Traceback (most recent call last):
  File "e:\LocalRepos\Projects\systra-semantic-search\systrasemsearch\sem_search_cli.py", line 249, in <module>
    local_file = pull_blob_file(local_storage, document_to_retrieve)
  File "e:\LocalRepos\Projects\systra-semantic-search\systrasemsearch\sem_search_cli.py", line 179, in pull_blob_file
    blob_data = blob.download_blob()
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\tracing\decorator.py", line 73, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_blob_client.py", line 868, in download_blob
    return StorageStreamDownloader(**options)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 355, in __init__
    self._response = self._initial_request()
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 435, in _initial_request
    process_storage_error(error)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_shared\response_handlers.py", line 181, in process_storage_error
    exec("raise error from None")   # pylint: disable=exec-used # nosec
  File "<string>", line 1, in <module>
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_download.py", line 392, in _initial_request
    location_mode, response = self._clients.blob.download(
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\tracing\decorator.py", line 73, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\storage\blob\_generated\operations\_blob_operations.py", line 1677, in download
    map_error(status_code=response.status_code, response=response, error_map=error_map)
  File "C:\Users\zepef\anaconda3\envs\systra\lib\site-packages\azure\core\exceptions.py", line 107, in map_error
    raise error
azure.core.exceptions.ResourceNotFoundError: The specified blob does not exist.
RequestId:0e7c2c3b-901e-0074-3b79-6aa9c6000000
Time:2022-05-18T05:36:32.2921373Z
ErrorCode:BlobNotFound
Content: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:0e7c2c3b-901e-0074-3b79-6aa9c6000000
Time:2022-05-18T05:36:32.2921373Z</Message></Error>

All the best !

Pierre-Emmanuel

**Comment by zepef** [comment]
Hello,

I have found something interesting. Remember my terminal output:

There are 16 documents in the 'azureblob-index' search index.
<Response [200]>
Please enter a query in French or English:
What is a knowledge graph?
en
Status code:  200
Please enter the response number:
0
Creating a Knowledge Graph with neo4j.pdf
C:\Users\Public\Documents\Creating a Knowledge Graph with neo4j.pdf Creating a Knowledge Graph with neo4j.pdf

When I implement a list blob function I only get 13 documents

    n = 1
    blob_list = container.list_blobs()
    for blob in blob_list:
        print(blob.name + ' - ' + str(n))
        n+=1

I found that the search index is not based on the real content of the container, the pdf found is no longer in it so it finds it in the query result but cannot load it because it is no longer there.

EUREKA, thanks for helping me eliminating all possible blob storage inconsistencies!

All the best,

Pierre-Emmanuel

De : vincenttran-msft ***@***.***>
Envoyé : mardi 17 mai 2022 23:29
À : Azure/azure-sdk-for-python ***@***.***>
Cc : Pierre-Emmanuel FEGA ***@***.***>; Mention ***@***.***>
Objet : Re: [Azure/azure-sdk-for-python] Cannot pull files from blobs from python anymore (Issue #24482)


Hi @zepef<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzepef&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=kao%2BKgWcP%2BE4vRX8eKH874sZ8rYiAGLvKM9FpXWDnMc%3D&reserved=0> Pierre-Emmanuel, unfortunately I was unable to reproduce your error as it seems to be working on my end. I think the first step is to try to clear up any ambiguity that might be the source of the error.

  1.  What exactly are you failing at (line or API call), and what is the exact error?
  2.  Just to confirm, you note that you cannot download "since azure-storage-blob==12.12.0.". If I am reading that correctly, what was the last version of azure-storage-blob that the operation worked for you?
  3.  For STORAGE_ACCOUNT_URL, please ensure it is in the following format: f"https://<storage_acc_name>.blob.core.windows.net<https://%3cstorage_acc_name%3e.blob.core.windows.net>"
  4.  For STORAGE_ACCOUNT_KEY, please ensure the key is correct and is still valid for your Storage account. Perhaps try rotating the key and entering the new key?
  5.  BLOB_CONTAINER_NAME should be a container that already exists on the Storage account
  6.  blob_filename should be a blob that already exists in the provided container (BLOB_CONTAINER_NAME)

Hopefully double-checking those values should help you resolve your issue. Please let me know if you have any additional follow-ups, and if possible, please provide a RequestID along with your future query. Thanks!
NOTE: When providing a RequestID, please give a more recent request's ID as the RequestIDs are automatically cleared on a rolling basis.

Hi @Bogdan-Stancescu-RBC<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBogdan-Stancescu-RBC&data=05%7C01%7C%7C829b63b0f25447eac41108da3843f15c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884161706488201%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dU8oM8hEoXemhlUqQvxKtaJKSFJIc8GegbtLyuKQGwE%3D&reserved=0> Bogdan, just as above, I was unable to reproduce your error as it seems to be working on my end. This seems to possibly be permission related, so please check your permissions (or consider using a different credential, if possible).
As a side-note, I attempted to reproduce your issue with azure-storage-blob=12.12.0 and azure-identity=1.10.0.
Furthermore, please ensure that myStorage resolves to your Storage account name properly.
If none of that is able to help resolve your issue, please attach a RequestID just as requested above for the other poster's query. Thanks!

-

**Comment by vincenttran-msft** [comment]
Hi @zepef Pierre-Emmanuel, you are very welcome! I am glad that we were able to work together to resolve your issue. If that is all, may I close the thread and mark your issue as resolved? Thanks!

Hi @Bogdan-Stancescu-RBC Bogdan, looking through our open issues, did you actually mean to respond to this thread? [https://github.com/Azure/azure-sdk-for-python/issues/24282](https://github.com/Azure/azure-sdk-for-python/issues/24282) It seems that your issue is more closely related to the linked thread than this current thread, and so if your issues are still persisting please post your follow-up in that thread so that we can further assist you. Thanks!

**Comment by zepef** [comment]
Hello,

You can close the thread and mark it as resolved thanks again !

¨Pierre-Emmanuel

De : vincenttran-msft ***@***.***>
Envoyé : mercredi 18 mai 2022 21:27
À : Azure/azure-sdk-for-python ***@***.***>
Cc : Pierre-Emmanuel FEGA ***@***.***>; Mention ***@***.***>
Objet : Re: [Azure/azure-sdk-for-python] Cannot pull files from blobs from python anymore (Issue #24482)


Hi @zepef<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzepef&data=05%7C01%7C%7Cb6c7281cdeec49449e8e08da38fc0552%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884952310688893%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=GGZjNQBuseWlFktIrHoZgorM7VDzDgiC4fOimePLWEE%3D&reserved=0> Pierre-Emmanuel, you are very welcome! I am glad that we were able to work together to resolve your issue. If that is all, may I close the thread and mark your issue as resolved? Thanks!

Hi @Bogdan-Stancescu-RBC<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FBogdan-Stancescu-RBC&data=05%7C01%7C%7Cb6c7281cdeec49449e8e08da38fc0552%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884952310688893%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PhZJcVg545QZM03DE%2F9fNSyf5KJdVbdjfl8qenbzcxE%3D&reserved=0> Bogdan, looking through our open issues, did you actually mean to respond to this thread? #24282<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-sdk-for-python%2Fissues%2F24282&data=05%7C01%7C%7Cb6c7281cdeec49449e8e08da38fc0552%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637884952310688893%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=OhDvliW%2BqJARpzCTIlUDckT5rjCnj4BdhmepmYiVvFY%3D&reserved=0> It seems that your issue is more closely related to the linked thread than this current thread, and so if your issues are still persisting please post your follow-up in that thread so that we can further assist you. Thanks!

-

**Comment by zepef** [comment]
Thanks again, now I am in deployment phase which is another interesting story...

All the best !

Pierre-Emmanuel

De : vincenttran-msft ***@***.***>
Envoyé : jeudi 19 mai 2022 21:06
À : Azure/azure-sdk-for-python ***@***.***>
Cc : Pierre-Emmanuel FEGA ***@***.***>; Mention ***@***.***>
Objet : Re: [Azure/azure-sdk-for-python] Cannot pull files from blobs from python anymore (Issue #24482)


Closed #24482<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure%2Fazure-sdk-for-python%2Fissues%2F24482&data=05%7C01%7C%7C0485469ed71a4c8665c208da39c23928%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637885803590413661%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=9ZKKiW1SQ0VOLuIa2ZsNQT7mnBn9Kb%2FB0zWA9JGaWNc%3D&reserved=0> as completed.

-
